pallets/assets/src/impl_fungibles.rs

```diff
@@ -46,7 +46,9 @@ impl<T: Config<I>, I: 'static> fungibles::Inspect<<T as SystemConfig>::AccountId
     }
 
     fn total_balance(asset: Self::AssetId, who: &<T as SystemConfig>::AccountId) -> Self::Balance {
-        Pallet::<T, I>::balance(asset, who)
+        let free_balance = Pallet::<T, I>::balance(asset.clone(), who);
+        let locked_balance = Locked::<T, I>::get(asset, who);
+        free_balance.saturating_add(locked_balance)
     }
```

Comment:
https://github.com/paritytech/polkadot-sdk/blob/8c8edacf8942298c3807a2e192860da9e7e4996a/substrate/frame/support/src/traits/tokens/fungible/regular.rs#L62-L80
should in `balance` function